tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
+  const mobileStaticPath = path.resolve(__dirname, '../../../client/static');
+
+  if (!fs.existsSync(`${mobileStaticPath}/mobile`)) {
+    fs.mkdir(`${mobileStaticPath}/mobile`, err => {
+      return err;
+    });
+    writeJson(json);
+  } else {
+    writeJson(json);
+  }
+
+  function writeJson(curriculum) {
+    curriculum = JSON.parse(curriculum);
+
+    const superBlocks = Object.keys(curriculum);
+
+    const superBlockObj = { superblocks: superBlocks.slice(1) };
+
+    writeToFile('availableSuperblocks', superBlockObj);
+
+    let blockNames = {};
+
+    for (let i = 0; i < superBlocks.length; i++) {
+      // blockGroup = object parent without key-name
+
+      const blockGroup = Object.keys(curriculum[superBlocks[i]].blocks);
```

Comment:
Shouldn't this be just `blockNames`?  It's an array of them, after all.  I think it's just a straight swap with the current `blockNames`!

Reply:
Not sure what you mean by this.

Reply:
Sorry: I meant that `blockGroup` has this structure

```
[
  'how-neural-networks-work',
  'machine-learning-with-python-projects',
  'tensorflow'
]
```
i.e. just an array of names.  So, it makes sense to call it `blockNames`.  That name is currently being used, though, so you'd need to rename the existing one.